### ISSUE TITLE
Hide Anaconda context menu entry on non-python projects

### ISSUE THREAD

**Issue by evandrocoan** [DamnWidget/anaconda#571]
### Hide Anaconda context menu entry on non-python projects

![image](https://cloud.githubusercontent.com/assets/5332158/19623717/3b603d7e-98b4-11e6-9f2e-999a2207cdc4.png)

## Expected Behaviour

To not see the `Anaconda` Context Menu entry is on non python projects.

## Actual Behaviour

The `Anaconda` Context Menu entry is showed on non python projects.

**Comment by DamnWidget** [comment]
I don't think we are able to do anything  beyond define the anaconda methods as disabled (that we already do).

**Comment by evandrocoan** [comment]
Well, there is a bad option which can be done while the issue https://github.com/SublimeTextIssues/Core/issues/1095 on the Sublime Text core is not closed.

When we are on no python projects, we delete the Context menu file.
When we are on python projects, we write the Context menu file down.
